CHANGELOG.md

```
@@ -1,5 +1,21 @@
 # Changelog
 
+## 1.17.6 - 2026-04-04
+
+#### 🐛 Bug Fixes
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)